packages/react-inbox/src/components/Message/index.tsx

```diff
@@ -44,7 +44,10 @@ const Message: React.FunctionComponent<IMessageProps> = ({
       3. from props.brand.inapp.icons.message
       4. from remote brand.inapp.icons.message
     */
-    icon ?? defaultIcon ?? !brand?.inapp?.disableMessageIcon
+    icon ||
+      brand?.inapp?.icons?.message ||
```

Comment:
`?.message` for icon? maybe I am not reading this right but it is bit hard to understand.